### DISCUSSION TITLE
Developer Mode

### DISCUSSION THREAD

**Discussion by tac900** [magicbug/Cloudlog#2029] [Q&A] [answered]
how do i get out of Developer Mode?  I also tried to copy the 2.4 files over top of the 2.3.  Do I need to restart the webserver?

**Comment by magicbug** [comment, marked as answer]
Edit the index.php environment variable

**Reply by AndreasK79** [reply]
Why aren't the releases shipped with this set as production?

**Reply by magicbug** [reply]
They probably will soon :)
